core/vm/eips.go

```diff
@@ -336,7 +339,7 @@ func opExtCodeCopyEIP4762(pc *uint64, interpreter *EVMInterpreter, scope *ScopeC
 		uint64CodeOffset = math.MaxUint64
 	}
 	addr := common.Address(a.Bytes20())
-	code := interpreter.evm.StateDB.GetCode(addr)
+	code := interpreter.evm.resolveCode(addr)
```

Comment:
Probably better to revert the change.

In context of verkle, the opExtCodeCopy should also act on the first two bytes of delegation

Reply:
done in 98521a50f8a95bd965aef258d5851079d829fee3